yugabyte/run-jepsen.py

```diff
@@ -89,6 +102,24 @@
 child_processes = []
 
 
+def get_workload_version(workload):
+    for el in TEST_PER_VERSION:
+        for tests in el["tests"]:
+            if workload in tests:
+                return el["start_version"]
+    assert False, f"Unanable to find workload in tests: {TESTS}"
+
+
+def compare_versions_less_than(v1, v2):
```

Comment:
I think this should be `is_versions_at_least(expected, actual)` (with inverted logic) because that's what we really care about.